lib/catalog/survey_compare.rb

```diff
@@ -0,0 +1,25 @@
+module Catalog
+  class SurveyCompare
+    attr_reader :base
+    class << self
+      def changed?(plan)
+        potential = new(plan)
+        encoded_base = Base64.strict_encode64(potential.base.to_json)
+        encoded_topo = Base64.strict_encode64(potential.topo_base.to_json)
+        encoded_topo != encoded_base
+      end
+    end
```

Comment:
I don't really mind since this isn't in `app/services` but it still behaves like a service class so I figured I'd ask the question: is there a reason we're not doing the "traditional" `.process.result` style thing for this "service"?.

Reply:
I placed this in `lib` due to the fact it acts like a validation class. There really is no obect to return since it only compares strings.

I do see that it could be abstracted more, and accept as args an object and two attributes to compare.

I don't see it as a service though.